praetorian_cli/ui/aegis/commands/schedule.py

```
@@ -22,6 +24,20 @@
 
 
+def _assume_schedule_account(menu, schedule_id):
+    """In multi-account mode, assume into the account owning this schedule."""
+    if not getattr(menu, 'multi_account_mode', False):
+        return True
+    acct_info = getattr(menu, 'schedule_account_map', {}).get(schedule_id, {})
+    acct_email = acct_info.get('account_email')
+    if acct_email:
+        try:
+            menu.sdk.accounts.assume_role(acct_email)
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)